src/test/java/io/confluent/connect/elasticsearch/bulk/BulkProcessorTest.java

```diff
@@ -62,15 +75,15 @@ private Expectation(List<Integer> request, BulkResponse response) {
     }
   }
 
-  private static final class Client implements BulkClient<Integer, List<Integer>> {
+  private static final class Client implements BulkClient<IndexableRecord, List<Integer>> {
```

Comment:
so that change was missed before here?

Reply:
i think the original author just didn't want to set up `IndexableRecords` 🤷